### ISSUE TITLE
[DEV] Exception: AttributeError: 'Pokemon' object has no attribute 'num'

### ISSUE THREAD

**Issue by ChrisHaPunkt** [PokemonGoF/PokemonGo-Bot#3766]
after latest dev pull:

> 2016-08-12 19:53:46,721 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Caterpie appeared! [CP 260] [Potential 0.16] [A/D/S 1/2/4]
2016-08-12 19:53:49,958 [PokemonCatchWorker] [INFO] [no_pokeballs] No usable pokeballs found!
2016-08-12 19:53:49,958 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Ultraball, with chance 47.87 (115 left)
2016-08-12 19:53:50,481 [       cli] [INFO]
2016-08-12 19:53:50,481 [       cli] [INFO] Ran for 0:02:13
2016-08-12 19:53:50,482 [       cli] [INFO] Total XP Earned: 2075  Average: 55957.62/h
2016-08-12 19:53:50,482 [       cli] [INFO] Travelled 0.00km
2016-08-12 19:53:50,482 [       cli] [INFO] Visited 0 stops
2016-08-12 19:53:50,482 [       cli] [INFO] Encountered 6 pokemon, 5 caught, 5 released, 2 evolved, 0 never seen before
2016-08-12 19:53:50,482 [       cli] [INFO] Threw 7 pokeballs
2016-08-12 19:53:50,482 [       cli] [INFO] Earned 500 Stardust
2016-08-12 19:53:50,482 [       cli] [INFO]
2016-08-12 19:53:50,483 [       cli] [INFO] Highest CP Pokemon: Nidoran F [CP: 589] [IV: 10/12/5] Potential: 0.6
2016-08-12 19:53:50,483 [       cli] [INFO] Most Perfect Pokemon: Nidoran F [CP: 589] [IV: 10/12/5] Potential: 0.6
Traceback (most recent call last):
  File "pokecli.py", line 602, in <module>
    main()
  File "pokecli.py", line 103, in main
    bot.tick()
  File "/opt/PokemonGo-Bot/pokemongo_bot/__init__.py", line 481, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "/opt/PokemonGo-Bot/pokemongo_bot/cell_workers/catch_visible_pokemon.py", line 49, in work
    self.catch_pokemon(self.bot.cell['catchable_pokemons'].pop(0))
  File "/opt/PokemonGo-Bot/pokemongo_bot/cell_workers/catch_visible_pokemon.py", line 70, in catch_pokemon
    return_value = worker.work()
  File "/opt/PokemonGo-Bot/pokemongo_bot/cell_workers/pokemon_catch_worker.py", line 102, in work
    self._do_catch(pokemon, encounter_id, catch_rate_by_ball, is_vip=is_vip)
  File "/opt/PokemonGo-Bot/pokemongo_bot/cell_workers/pokemon_catch_worker.py", line 370, in _do_catch
    'pokemon_id': pokemon.num
AttributeError: 'Pokemon' object has no attribute 'num'

### Your config.json (remove your credentials and any other private info)
```
{
  "auth_service": "google",
  "username": "XXXX",
  "password": "XX",
  "location": "-37.832370, 144.9812770",
  "gmapkey": "XX",
  "libencrypt_location": "./",
  "health_record": false,
  "tasks": [
    {
      "type": "HandleSoftBan"
    },
    {
      "type": "CollectLevelUpReward"
    },
    {
      "type": "IncubateEggs",
      "config": {
        "longer_eggs_first": true
      }
    },
    {
      "type": "EvolvePokemon",
      "config": {
        "evolve_all": "Pidgey, Weedle, Caterpie, Rattata",
        "first_evolve_by": "cp",
        "evolve_above_cp": 1,
        "evolve_above_iv": 0.1,
        "logic": "or",
        "evolve_speed": 15,
        "use_lucky_egg": false
      }
    },
    {
      "type": "TransferPokemon"
    },
    {
      "type": "RecycleItems",
      "config": {
        "min_empty_space": 15,
        "item_filter": {
          "Pokeball": {
            "keep": 5
          },
          "Greatball": {
            "keep": 5
          },
          "Ultraball": {
            "keep": 130
          },
          "Potion": {
            "keep": 1
          },
          "Super Potion": {
            "keep": 1
          },
          "Hyper Potion": {
            "keep": 1
          },
          "Max Potion": {
            "keep": 80
          },
          "Revive": {
            "keep": 0
          },
          "Max Revive": {
            "keep": 30
          },
          "Razz Berry": {
            "keep": 40
          }
        }
      }
    },
    {
      "type": "NicknamePokemon",
      "config": {
        "dont_nickname_favorite": true,
        "nickname_template": ""
      }
    },
    {
      "type": "CatchVisiblePokemon"
    },
    {
      "type": "CatchLuredPokemon"
    },
    {
      "type": "SpinFort"
    },
    {
      "type": "MoveToMapPokemon",
      "config": {
        "address": "cxcyxcyxc",
        "max_distance": 500,
        "min_time": 60,
        "min_ball": 50,
        "prioritize_vips": true,
        "snipe": true,
        "snipe_high_prio_only": true,
        "snipe_high_prio_threshold": 400,
        "update_map": true,
        "mode": "priority",
        "map_path": "raw_data",
        "catch": {
          "==========Legendaries==========": 0,
          "Aerodactyl": 1000,
          "Snorlax": 1000,
          "Articuno": 1000,
          "Zapdos": 1000,
          "Moltres": 1000,
          "Dratini": 1000,
          "Dragonair": 1000,
          "Dragonite": 1000,
          "Mewtwo": 1000,
          "Mew": 1000,
          "==========Region Locked==========": 0,
          "Farfetch'd": 1000,
          "Kangaskhan": 1000,
          "Mr. Mime": 1000,
          "Tauros": 1000,
          "==========Very Rare==========": 0,
          "Lapras": 900,
          "Electabuzz": 900,
          "Magmar": 900,
          "Ditto": 900,
          "==========Starters==========": 0,
          "Bulbasaur": 400,
          "Ivysaur": 600,
          "Venusaur": 1000,
          "Charmander": 400,
          "Charmeleon": 600,
          "Charizard": 1000,
          "Squirtle": 400,
          "Wartortle": 600,
          "Blastoise": 1000,
          "Pikachu": 600,
          "Raichu": 1000,
          "==========Semi Rare==========": 0,
          "Porygon": 200,
          "Scyther": 200,
          "Jynx": 200,
          "==========Uncommon==========": 0,
          "Omanyte": 150,
          "Omastar": 500,
          "Seel": 300,
          "Dewgong": 500,
          "Grimer": 200,
          "Muk": 500,
          "Shellder": 200,
          "Cloyster": 500,
          "Gastly": 200,
          "Haunter": 500,
          "Gengar": 1000,
          "Onix": 600,
          "Drowzee": 600,
          "Hypno": 600,
          "Vulpix": 200,
          "Ninetales": 600,
          "Paras": 100,
          "Parasect": 500,
          "Growlithe": 200,
          "Arcanine": 700,
          "Tentacool": 200,
          "Tentacruel": 500,
          "Mankey": 150,
          "Primeape": 500,
          "Clefairy": 150,
          "Clefable": 500,
          "Jigglypuff": 150,
          "Wigglytuff": 500,
          "Venonat": 100,
          "Venomoth": 500,
          "Diglett": 200,
          "Dugtrio": 500,
          "Meowth": 250,
          "Persian": 500,
          "Psyduck": 150,
          "Golduck": 500,
          "Geodude": 100,
          "Graveler": 500,
          "Golem": 800,
          "Eevee": 200,
          "Vaporeon": 800,
          "Jolteon": 800,
          "Flareon": 800,
          "Kabuto": 150,
          "Kabutops": 500,
          "Magikarp": 150,
          "Gyarados": 800,
          "Pinsir": 150,
          "Ponyta": 200,
          "Rapidash": 500,
          "Slowpoke": 200,
          "Slowbro": 500,
          "Magnemite": 250,
          "Magneton": 500,
          "Krabby": 100,
          "Kingler": 500,
          "Voltorb": 200,
          "Electrode": 500,
          "Exeggcute": 250,
          "Exeggcutor": 500,
          "Cubone": 300,
          "Marowak": 800,
          "Hitmonlee": 400,
          "Hitmonchan": 400,
          "Lickitung": 500,
          "Koffing": 200,
          "Weezing": 500,
          "Rhyhorn": 200,
          "Rhydon": 500,
          "Chansey": 800,
          "Tangela": 300,
          "Horsea": 200,
          "Seadra": 600,
          "Goldeen": 150,
          "Seaking": 500,
          "Staryu": 200,
          "Starmie": 800,
          "==========T1 Evolvers==========": 0,
          "Caterpie": 10,
          "Metapod": 10,
          "Butterfree": 500,
          "Weedle": 10,
          "Kakuna": 10,
          "Beedrill": 500,
          "Pidgey": 10,
          "Pidgeotto": 10,
          "Pidgeot": 300,
          "==========T2 Evolvers==========": 0,
          "Nidoran F": 10,
          "Nidorina": 10,
          "Nidoqueen": 10,
          "Nidoran M": 10,
          "Nidorino": 10,
          "Nidoking": 10,
          "Oddish": 100,
          "Gloom": 200,
          "Vileplume": 600,
          "Poliwag": 200,
          "Poliwhirl": 400,
          "Poliwrath": 800,
          "Abra": 300,
          "Kadabra": 600,
          "Alakazam": 800,
          "Machop": 150,
          "Machoke": 400,
          "Machamp": 800,
          "Bellsprout": 100,
          "Weepinbell": 400,
          "Victreebel": 800,
          "==========Trash==========": 0,
          "Rattata": 10,
          "Raticate": 10,
          "Spearow": 10,
          "Fearow": 10,
          "Ekans": 10,
          "Arbok": 10,
          "Sandshrew": 10,
          "Sandslash": 10,
          "Zubat": 10,
          "Golbat": 10,
          "Doduo": 10,
          "Dodrio": 10
        }
      }
    },
    {
      "type": "MoveToFort",
      "config": {
        "lure_attraction": true,
        "lure_max_distance": 2000
      }
    },
    {
      "type": "FollowSpiral",
      "config": {
        "diameter": 16,
        "step_size": 70
      }
    }
  ],
  "map_object_cache_time": 5,
  "forts": {
    "avoid_circles": false,
    "max_circle_size": 100
  },
  "walk": 13.16,
  "action_wait_min": 1,
  "action_wait_max": 4,
  "debug": false,
  "test": false,
  "location_cache": false,
  "distance_unit": "km",
  "reconnecting_timeout": 15,
  "catch_randomize_reticle_factor": 1.0,
  "catch_randomize_spin_factor": 1.0,
  "min_ultraball_to_keep": 10,
  "catch": {
    "any": {
      "catch_above_cp": 0,
      "catch_above_iv": 0,
      "logic": "or"
    },
    "// Pokemons with example": {
      "always_catch": true
    },
    "// Gets filtered with release parameters": {},
    "// Legendary pokemons (Goes under S-Tier)": {},
    "Lapras": {
      "always_catch": true
    },
    "Moltres": {
      "always_catch": true
    },
    "Zapdos": {
      "always_catch": true
    },
    "Articuno": {
      "always_catch": true
    },
    "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
    "Mewtwo": {
      "always_catch": true
    },
    "Dragonite": {
      "always_catch": true
    },
    "Snorlax": {
      "always_catch": true
    },
    "// Mew evolves to Mewtwo": {},
    "Mew": {
      "always_catch": true
    },
    "Arcanine": {
      "always_catch": true
    },
    "Vaporeon": {
      "always_catch": true
    },
    "Gyarados": {
      "always_catch": true
    },
    "Exeggutor": {
      "always_catch": true
    },
    "Muk": {
      "always_catch": true
    },
    "Weezing": {
      "always_catch": true
    },
    "Flareon": {
      "always_catch": true
    },
    "// Growlithe evolves to Arcanine": {},
    "Growlithe": {
      "always_catch": true
    },
    "// Dragonair evolves to Dragonite": {},
    "Dragonair": {
      "always_catch": true
    },
    "// Grimer evolves to Muk": {},
    "Grimer": {
      "always_catch": true
    },
    "// Magikarp evolves to Gyarados": {},
    "Magikarp": {
      "always_catch": true
    },
    "// Exeggcute evolves to Exeggutor": {},
    "Exeggcute": {
      "always_catch": true
    },
    "// Eevee evolves to many versions, like Vaporeon, Flareon": {},
    "Eevee": {
      "always_catch": true
    },
    "// A-Tier pokemons": {},
    "Slowbro": {
      "always_catch": true
    },
    "Victreebel": {
      "always_catch": true
    },
    "Machamp": {
      "always_catch": true
    },
    "Poliwrath": {
      "always_catch": true
    },
    "Clefable": {
      "always_catch": true
    },
    "Nidoking": {
      "always_catch": true
    },
    "Venusaur": {
      "always_catch": true
    },
    "Charizard": {
      "always_catch": true
    },
    "Golduck": {
      "always_catch": true
    },
    "Nidoqueen": {
      "always_catch": true
    },
    "Vileplume": {
      "always_catch": true
    },
    "Blastoise": {
      "always_catch": true
    },
    "Omastar": {
      "always_catch": true
    },
    "Aerodactyl": {
      "always_catch": true
    },
    "Golem": {
      "always_catch": true
    },
    "Wigglytuff": {
      "always_catch": true
    },
    "Dewgong": {
      "always_catch": true
    },
    "Ninetales": {
      "always_catch": true
    },
    "Magmar": {
      "always_catch": true
    },
    "Kabutops": {
      "always_catch": true
    },
    "Electabuzz": {
      "always_catch": true
    },
    "Starmie": {
      "always_catch": true
    },
    "Jolteon": {
      "always_catch": true
    },
    "Rapidash": {
      "always_catch": true
    },
    "Pinsir": {
      "always_catch": true
    },
    "Scyther": {
      "always_catch": true
    },
    "Tentacruel": {
      "always_catch": true
    },
    "Gengar": {
      "always_catch": true
    },
    "Hypno": {
      "always_catch": true
    },
    "Pidgeot": {
      "always_catch": true
    },
    "Rhydon": {
      "always_catch": true
    },
    "Seaking": {
      "always_catch": true
    },
    "Kangaskhan": {
      "always_catch": true
    }
  },
  "release": {
    "any": {
      "release_below_cp": 800,
      "release_below_iv": 0.8,
      "logic": "or",
      "keep_best_iv": 1,
      "keep_best_cp": 1
    },
    "// Legendary pokemons (Goes under S-Tier)": {},
    "Lapras": {
      "release_below_cp": 1041,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Moltres": {
      "release_below_cp": 1132,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Zapdos": {
      "release_below_cp": 1087,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Articuno": {
      "release_below_cp": 1039,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
    "Mewtwo": {
      "release_below_cp": 1447,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Dragonite": {
      "release_below_cp": 1221,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Snorlax": {
      "release_below_cp": 2417,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Mew evolves to Mewtwo": {},
    "Mew": {
      "release_below_cp": 1152,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Arcanine": {
      "release_below_cp": 1041,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Vaporeon": {
      "release_below_cp": 2020,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Gyarados": {
      "release_below_cp": 2206,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Exeggutor": {
      "release_below_cp": 2580,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Muk": {
      "release_below_cp": 909,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Weezing": {
      "release_below_cp": 784,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Flareon": {
      "release_below_cp": 924,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Growlithe evolves to Arcanine": {},
    "Growlithe": {
      "release_below_cp": 958,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Dragonair evolves to Dragonite": {},
    "Dragonair": {
      "release_below_cp": 958,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Dratini": {
      "release_below_cp": 750,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Grimer evolves to Muk": {},
    "Grimer": {
      "release_below_cp": 892,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "// Magikarp evolves to Gyarados": {},
    "Magikarp": {
      "release_below_cp": 214,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "// Exeggcute evolves to Exeggutor": {},
    "Exeggcute": {
      "release_below_cp": 881,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "// Eevee evolves to many versions, like Vaporeon, Flareon": {},
    "Eevee": {
      "release_below_cp": 822,
      "release_below_iv": 0.8,
      "keep_best_cp": 2,
      "logic": "or"
    },
    "// A-Tier pokemons": {},
    "Slowbro": {
      "release_below_cp": 1979,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Victreebel": {
      "release_below_cp": 883,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Machamp": {
      "release_below_cp": 907,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Poliwrath": {
      "release_below_cp": 1893,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Clefable": {
      "release_below_cp": 1866,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Nidoking": {
      "release_below_cp": 864,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Venusaur": {
      "release_below_cp": 902,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Charizard": {
      "release_below_cp": 909,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Golduck": {
      "release_below_cp": 1725,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Nidoqueen": {
      "release_below_cp": 868,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Vileplume": {
      "release_below_cp": 871,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Blastoise": {
      "release_below_cp": 888,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Omastar": {
      "release_below_cp": 780,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Aerodactyl": {
      "release_below_cp": 756,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Golem": {
      "release_below_cp": 804,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Wigglytuff": {
      "release_below_cp": 760,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Dewgong": {
      "release_below_cp": 748,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Ninetales": {
      "release_below_cp": 763,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Magmar": {
      "release_below_cp": 1369,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Kabutops": {
      "release_below_cp": 744,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Electabuzz": {
      "release_below_cp": 1399,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Starmie": {
      "release_below_cp": 1449,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Jolteon": {
      "release_below_cp": 746,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Rapidash": {
      "release_below_cp": 768,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Pinsir": {
      "release_below_cp": 1771,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Scyther": {
      "release_below_cp": 1537,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Tentacruel": {
      "release_below_cp": 1209,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Gengar": {
      "release_below_cp": 724,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Hypno": {
      "release_below_cp": 763,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Pidgeot": {
      "release_below_cp": 1441,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Rhydon": {
      "release_below_cp": 1725,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Seaking": {
      "release_below_cp": 1378,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Kangaskhan": {
      "release_below_cp": 712,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Koffing evolves to Weezing (A-Tier)": {},
    "Koffing": {
      "release_below_cp": 875,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "// Below is B-tier and lower pokemons": {},
    "Caterpie": {
      "release_below_cp": 300,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Weedle": {
      "release_below_cp": 328,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Diglett": {
      "release_below_cp": 301,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Metapod": {
      "release_below_cp": 264,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Kakuna": {
      "release_below_cp": 265,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Rattata": {
      "release_below_cp": 550,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Abra": {
      "release_below_cp": 366,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Zubat": {
      "release_below_cp": 600,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Chansey": {
      "release_below_cp": 235,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Pidgey": {
      "release_below_cp": 650,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Spearow": {
      "release_below_cp": 538,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Meowth": {
      "release_below_cp": 573,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Krabby": {
      "release_below_cp": 580,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Sandshrew": {
      "release_below_cp": 441,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Poliwag": {
      "release_below_cp": 590,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Horsea": {
      "release_below_cp": 624,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Gastly": {
      "release_below_cp": 561,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Ekans": {
      "release_below_cp": 602,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Shellder": {
      "release_below_cp": 457,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Vulpix": {
      "release_below_cp": 467,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Voltorb": {
      "release_below_cp": 583,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Geodude": {
      "release_below_cp": 600,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Doduo": {
      "release_below_cp": 608,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Onix": {
      "release_below_cp": 491,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Mankey": {
      "release_below_cp": 644,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Pikachu": {
      "release_below_cp": 496,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Magnemite": {
      "release_below_cp": 750,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Tentacool": {
      "release_below_cp": 677,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Paras": {
      "release_below_cp": 668,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Jigglypuff": {
      "release_below_cp": 704,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Ditto": {
      "release_below_cp": 321,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Staryu": {
      "release_below_cp": 756,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Charmander": {
      "release_below_cp": 680,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Goldeen": {
      "release_below_cp": 801,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Squirtle": {
      "release_below_cp": 800,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Cubone": {
      "release_below_cp": 561,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Venonat": {
      "release_below_cp": 807,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Bulbasaur": {
      "release_below_cp": 400,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Drowzee": {
      "release_below_cp": 676,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Machop": {
      "release_below_cp": 845,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Seel": {
      "release_below_cp": 717,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Kabuto": {
      "release_below_cp": 698,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Psyduck": {
      "release_below_cp": 623,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Bellsprout": {
      "release_below_cp": 897,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Omanyte": {
      "release_below_cp": 710,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Kadabra": {
      "release_below_cp": 387,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Oddish": {
      "release_below_cp": 689,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Dugtrio": {
      "release_below_cp": 408,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Rhyhorn": {
      "release_below_cp": 871,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Clefairy": {
      "release_below_cp": 923,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Slowpoke": {
      "release_below_cp": 945,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Pidgeotto": {
      "release_below_cp": 689,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Farfetch'd": {
      "release_below_cp": 441,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Poliwhirl": {
      "release_below_cp": 1005,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Nidoran M": {
      "release_below_cp": 700,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Nidoran F": {
      "release_below_cp": 700,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Nidorino": {
      "release_below_cp": 858,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Haunter": {
      "release_below_cp": 513,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Nidorina": {
      "release_below_cp": 1039,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Graveler": {
      "release_below_cp": 903,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Beedrill": {
      "release_below_cp": 850,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Raticate": {
      "release_below_cp": 1200,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Butterfree": {
      "release_below_cp": 508,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Hitmonlee": {
      "release_below_cp": 520,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Ponyta": {
      "release_below_cp": 934,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Hitmonchan": {
      "release_below_cp": 530,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Charmeleon": {
      "release_below_cp": 544,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Wartortle": {
      "release_below_cp": 738,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Persian": {
      "release_below_cp": 568,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Lickitung": {
      "release_below_cp": 568,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Ivysaur": {
      "release_below_cp": 571,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Electrode": {
      "release_below_cp": 880,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Marowak": {
      "release_below_cp": 578,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Gloom": {
      "release_below_cp": 1072,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Porygon": {
      "release_below_cp": 590,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Seadra": {
      "release_below_cp": 1262,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Jynx": {
      "release_below_cp": 1041,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Weepinbell": {
      "release_below_cp": 602,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Tangela": {
      "release_below_cp": 1395,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Fearow": {
      "release_below_cp": 1040,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Parasect": {
      "release_below_cp": 724,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Machoke": {
      "release_below_cp": 910,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Arbok": {
      "release_below_cp": 616,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Sandslash": {
      "release_below_cp": 1213,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Alakazam": {
      "release_below_cp": 633,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Kingler": {
      "release_below_cp": 962,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Dodrio": {
      "release_below_cp": 1372,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Tauros": {
      "release_below_cp": 643,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Primeape": {
      "release_below_cp": 650,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Magneton": {
      "release_below_cp": 657,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Venomoth": {
      "release_below_cp": 660,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Golbat": {
      "release_below_cp": 1539,
      "release_below_iv": 0.8,
      "logic": "or"
    },
    "Raichu": {
      "release_below_cp": 708,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Cloyster": {
      "release_below_cp": 717,
      "release_below_iv": 0.8,
      "logic": "and"
    },
    "Mr. Mime": {
      "release_below_cp": 650,
      "release_below_iv": 0.8,
      "logic": "and"
    }
  },
  "vips": {
    "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate!": {},
    "any": {
      "catch_above_cp": 1200,
      "catch_above_iv": 0.9,
      "logic": "or"
    },
    "Lapras": {},
    "Moltres": {},
    "Zapdos": {},
    "Articuno": {},
    "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
    "Mewtwo": {},
    "Dragonite": {},
    "Snorlax": {},
    "// Mew evolves to Mewtwo": {},
    "Mew": {},
    "Arcanine": {},
    "Vaporeon": {},
    "Gyarados": {},
    "Exeggutor": {},
    "Muk": {},
    "Weezing": {},
    "Flareon": {},
    "Charmander": {},
    "Charmeleon": {}
  }
}

```

### Other Information
OS: deb8
Branch: dev
Git Commit:4b632d84c2c4422ee0cd0342b2b882e8c1bce2bb
Python Version: 2.7

**Comment by zx900930** [comment]
same here with dev commit: 5a8a95a ("Writing the location file to fix the web ui (#3767)", 2016-08-12)

**Comment by Foulwerp** [comment]
It's line 370 in pokemon_catch_worker.py

its pokemon.num when it should be pokemon.pokemon_id

```
                        'pokemon_id': pokemon.num
```

```
                        'pokemon_id': pokemon.pokemon_id
```

**Comment by ynvtlmr** [comment]
replacing **pokemon.num** with **pokemon.pokemon_id** appears to work as intended.

**Comment by BriceSD** [comment]
pokemon.id actualy

**Comment by Foulwerp** [comment]
Well picked the wrong thumbs direction my bad.

**Comment by BriceSD** [comment]
pokemon.id actualy 
```
        # Unique ID for this particular Pokemon
        self.id = data.get('id', 0)
        # Id of the such pokemons in pokedex
        self.pokemon_id = data['pokemon_id']
```

**Comment by BriceSD** [comment]
It’s been fixed. Please close the issue

**Comment by k4n30** [comment]
Fixed by #3780
